_episodes/01-software-development-intro.md

```diff
@@ -50,60 +47,93 @@ smaller sub-modules. Another examples of modules are classes in object-oriented
 > it returns as an output. 
 > 
 {: .callout}  
-        
+
+There are various software architectures around. [Model–View–Controller (MVC) architecture](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) is one way to define 
+how the code is divided into smaller modules with well defined roles; other examples include 
+[Service-Oriented Architecture (SOA)](https://en.wikipedia.org/wiki/Service-oriented_architecture), 
+[Client-Server architecture](https://en.wikipedia.org/wiki/Client%E2%80%93server_model), 
+[N-tier architecture](https://en.wikipedia.org/wiki/Multitier_architecture), etc.
```

Comment:
Stating other examples is great, but I think adding links without adding explanations for what they are may be a bit confusing and jargony for learners. I'd either remove them, or perhaps add single sentence descriptions for where they are useful/when they are used, or bullet-point them and have a one-sentence explanation for each?

Reply:
This was addressed separately via 471b60750444f80381483ccd5a4448bb635a3e58